Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2843,19 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
+     * the admin screen.
+     */
+    public final function setIsShowMosaicButton($isShown)
```

Comment:
You renamed it to `setShow…` I would like just `show…` better, what do you think ?